plasmapy/dispersion/tests/test_dispersion.py

```diff
@@ -196,11 +196,9 @@ def test_plasma_dispersion_func_roots(self):
 
         for root in roots:
             Z_at_root = plasma_dispersion_func(root)
-            assert np.isclose(Z_at_root, 0 + 0j, atol=1e-15 * (1 + 1j)), (
-                "A root of the plasma dispersion function is expected at w = "
-                f"{root}, but plasma_dispersion_func({root}) is equal to "
-                f"{Z_at_root} instead of {0j}."
-            )
+            assert np.isclose(
+                Z_at_root, 0 + 0j, atol=1e-15 * (1 + 1j)
+            ), f"A root of the plasma dispersion function is expected at w = {root}, but plasma_dispersion_func({root}) is equal to {Z_at_root} instead of 0j."
```

Comment:
```suggestion
            ), (
                "A root of the plasma dispersion function is expected "
                f"at w = {root}, but plasma_dispersion_func({root}) is "
                f"equal to {Z_at_root} instead of 0j."
            )
```